data/json/items/comestibles/veggy_dishes.json

```diff
@@ -107,7 +107,8 @@
     "healthy": 2,
     "description": "A serving of cooked buckwheat groats.  Healthy and nutritious but bland.",
     "price": 120,
-    "flags": [ "EATEN_HOT" ]
+    "flags": [ "EATEN_HOT" ],
```

Comment:
I must argue that buckwheat is better cold, especially with the milk.

Reply:
Bah, personal preferences, I for one used to like my cooked cereals cold, then hot, and now lukewarm.